afl/src/lib.rs

```diff
@@ -68,10 +70,15 @@ where
 
     let mut input = vec![];
 
+    let loop_count: usize = env::var("AFL_FUZZER_LOOPCOUNT")
+        .unwrap_or_else(|_| u32::MAX.to_string())
+        .parse::<usize>()
+        .expect("Failed to parse environment variable to a number");
```

Comment:
```suggestion
    let loop_count = if let Ok(value) = env::var("AFL_FUZZER_LOOPCOUNT") {
        value
            .parse()
            .expect("Failed to parse environment variable to a number")
    } else {
        usize::MAX
    };      
```
Does this look okay to you?